src/extensions/replay/sessionrecording.ts

```diff
@@ -99,6 +99,15 @@ interface SnapshotBuffer {
     add(properties: Properties): void
 }
 
+/**
+ * local storage and session storage are limited to 5MB per origin (browsers vary but chrome is 5mb)
+ * easy for a full snapshot to be this larg on some sites so we would break even other use of local storage by the SDK
```

Comment:
```suggestion
 * easy for a full snapshot to be this large on some sites so we would even break other uses of local storage by the SDK
```

Reply:
oh 🙈 that's not supposed to be there 🤣

Reply:
was gonna say that, seems like a stash leaking around :D